### ISSUE TITLE
Batch size 1

### ISSUE THREAD

**Issue by matthew-jurewicz** [lucidrains/stylegan2-pytorch#115]
Can't find batch normalization in the code, so presumably this works just fine with batch size 1?
